app/frontend/src/features/dashboard/constants.ts

```diff
@@ -21,7 +21,7 @@ export const LOAD_MORE = "Load more";
 
 // Copy
 export const WELCOME = "Welcome to the Couchers.org Beta!";
-export const WEEKLY_EVENTS_TITLE = "Weekly online events";
+export const WEEKLY_EVENTS_TITLE = "Weekly Online Events";
```

Comment:
Is this capitalisation on every word on this only? Or are we meant to be doing this in other places?

Reply:
just on the dashboard titles in this case